lib/utils/cert-utils.js

```diff
@@ -0,0 +1,103 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs-extra');
+const forge = require('node-forge');
+const logger = require('@blackbaud/skyux-logger');
+
+const certCommonName = 'SKYUX-Developer-Certificate';
+const certName = 'skyux-server.pem';
+const keyName = 'skyux-server.key';
+
+const defaultCertRootPath = path.resolve(`${os.homedir()}/.skyux/certs/`);
+const defaultCertPath = path.join(defaultCertRootPath, certName);
+const defaultKeyPath = path.join(defaultCertRootPath, keyName);
+
+function error(msg) {
+  logger.error(msg);
+  return false;
+}
+
+function validate(argv) {
+  const cert = getCertPath(argv);
+  const key = getKeyPath(argv);
+
+  if (!fs.existsSync(cert)) {
+    return error(`Error locating certificate: ${cert}`);
+  }
+
+  if (!fs.existsSync(key)) {
+    return error(`Error locating key: ${key}`);
+  }
+
+  // TODO: VALIDATE CERT + KEY using forge
```

Comment:
Just confirming that we want to wait to implement this? If so, would it be worth creating an issue so we don't forget?

Reply:
I think it would be nice to implement, but I was concerned with timing on finding the exact command to run.  Filling issue as https://github.com/blackbaud/skyux-sdk-cli/issues/37